modules/dnn/src/onnx/onnx_graph_simplifier.cpp

```diff
@@ -145,13 +156,266 @@ class SoftMaxSubgraph : public Subgraph
     int axis;
 };
 
+class FuseUpsampleSubgraph : public Subgraph
+{
+public:
+    FuseUpsampleSubgraph() : scaleH(1), scaleW(1), emptyNodeId(-1) {}
+
+    virtual bool match(const Ptr<ImportGraphWrapper>& net, int nodeId,
+                    std::vector<int>& matchedNodesIds,
+                    std::vector<int>& targetNodesIds) CV_OVERRIDE
+    {
+        return Subgraph::match(net, nodeId, matchedNodesIds, targetNodesIds);
+    }
```

Comment:
Do not override this method if it just calls partent's implementation

Reply:
okay